docs/developer-guide/serialization.rst

```diff
@@ -365,5 +369,48 @@ the ``WRAP_SINGLE_VALUE`` is set to ``false``, for example:
 
 For more information, see :ref:`ksql_single_field_wrapping`.
 
-===========================
 Field Name Case Sensitivity
+---------------------------
+
+The format is case-insensitive when matching a KSQL field name with an Avro record's field name.
+The first case-insensitive match is used.
+
+.. _kafka_format
+
+-----
+KAFKA
+-----
+
+The ``KAFKA`` format supports``INT``, ``BIGINT``, ``DOUBLE`` and ``STRING`` primitives that have
+been serialized using Kafka's standard set of serializers.
+
+The format is designed primarily to support primitive message keys. It can be used as a value format,
+though certain operations are not yet supported when this is the case.
+
+Unlike some other formats, this format does not perform any type coercion. It is therefore important
+to correctly match the field type to the underlying serialized form to avoid deserialization errors.
+
++-------------------------+--------------------------------------------------------------------------------------------+
+| KSQL Field Type         | Kafka Type                                                                                 |
++=========================+============================================================================================+
+| INT / INTEGER           | A 32-bit signed integer, as has been serialized using the Java                             |
```

Comment:
Maybe: "A 32-bit signed integer, that was serialized by using the Java..."
Or: "A 32-bit signed integer serialized by using the Java..."